### PR TITLE
chore: add src folder

### PR DESCRIPTION
**Pull request by filipesilva**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ with excellent support for Angular apps that use routing.
 
 Here are the test related scripts:
 * `npm test` - compiles, runs and watches the karma unit tests
-* `npm run e2e` - run protractor e2e tests, written in JavaScript (*e2e-spec.js)
+* `npm run e2e` - compiles and run protractor e2e tests, written in JavaScript (*e2e-spec.js)
```
Comment (Foxandxss) [inline README.md]: Aren't them typescript now?

File: README.md [hunk 2]
```diff
@@ -128,9 +128,9 @@ These tools are configured for specific conventions described below.
 We recommend that you shut down one before starting another.*
 
 ### Unit Tests
-TypeScript unit-tests are usually in the `app` folder. Their filenames must end in `.spec`.
+TypeScript unit-tests are usually in the `src/app` folder. Their filenames must end in `.spec`.
```
Comment (Foxandxss) [inline README.md]: Shouldn't they end on `.spec.ts`?

File: README.md [hunk 3]
```diff
@@ -156,7 +156,7 @@ we configured protractor to find them.
 
 Thereafter, run them with `npm run e2e`.
 
-That command first compiles, then simultaneously starts the Http-Server at `localhost:8080`
+That command first compiles, then simultaneously starts the Http-Server at `localhost:3000`
```
Comment (Foxandxss) [inline README.md]: it is not `http-server` anymore, even if lite-server is a http server, I guess we could remove the - in there.

File: README.md [hunk 4]
```diff
@@ -147,7 +147,7 @@ restart it. No worries; it's pretty quick.
 
 ### End-to-end (E2E) Tests
 
-E2E tests are in the `e2e` directory, side by side with the `app` folder.
+E2E tests are in the `e2e` directory, side by side with the `src` folder.
```
Comment (Foxandxss) [inline README.md]: I see them in an `e2e` folder and still same folder after this PR.

File: README.md (hunk 1 above)
Comment (filipesilva) [inline README.md]: Yes, and that means they need to be compiled. However what I actually wanted to say here (and didn't do it very well) is that running the command also compiles your app.

File: README.md (hunk 2 above)
Comment (filipesilva) [inline README.md]: Fixed

File: README.md (hunk 1 above)
Comment (filipesilva) [inline README.md]: Sorry I missed what you really meant. Fixed.

File: README.md (hunk 4 above)
Comment (filipesilva) [inline README.md]: The paragraph says they are in the `e2e` folder, and that it is a sibling folder to `src`. Did you misread or am I missing something?

File: README.md (hunk 3 above)
Comment (filipesilva) [inline README.md]: Fixed by putting the package name explicitly.

File: README.md (hunk 4 above)
Comment (Foxandxss) [inline README.md]: I completely misread. Sorry

File: src/systemjs.config.js [hunk 5]
```diff
@@ -30,7 +30,6 @@
     // packages tells the System loader how to load when no filename and/or no extension
     packages: {
       app: {
-        main: './main.js',
```
Comment (Foxandxss) [inline src/systemjs.config.js]: what do we gain by removing this? I mean, if we are not importing "app" anymore, shouldn't we remove the entire app node?

Comment (filipesilva) [inline src/systemjs.config.js]: Doesn't work without that:
image

**Pull request merged**